core/src/main/java/discord4j/core/object/entity/Webhook.java

```diff
@@ -179,8 +180,8 @@ public Snowflake getCreatorId() {
      *
      * @return The secure token of this webhook.
      */
-    public String getToken() {
-        return data.token().get(); // TODO FIXME: really Possible?
+    public Optional<String> getToken() {
+        return data.token().toOptional();
```

Comment:
It is possible for this to be missing, I was not sure it was appropriate to return possible tho.

Reply:
What determines if it is present or not?

Reply:
I believe it is returned if the bot has the MANAGE_WEBHOOKS permission. I am not completely sure tho